tests/Metadata/Driver/DoctrineDriverTest.php

```diff
@@ -24,6 +25,10 @@ public function getMetadata()
 
     public function testMetadataForEmbedded()
     {
+        if (ORMVersion::compare('2.5')) {
```

Comment:
I guess here should be `ORMVersion::compare('2.5')>=0`

> 	Returns -1 if older, 0 if it is the same, 1 if version passed as argument is newer.

Returns -1 if older, 0 if it is the same, 1 if version passed as argument is newer.
--

Reply:
```suggestion
        if (ORMVersion::compare('2.5')>=0) {
```